tiledb/sm/config/config.h

```diff
@@ -233,6 +233,9 @@ class Config {
   /** Which reader to use for sparse global order queries. */
   static const std::string SM_QUERY_SPARSE_GLOBAL_ORDER_READER;
 
+  /** If `true`, run preprocessing merge step in sparse global order reader */
```

Comment:
Let's use `if nonzero` here as well and explain that it specifies "the amount of work per parallel task."

Reply:
9c9a5f72f